00-open-a-shapefile.Rmd

```diff
@@ -72,16 +72,15 @@ Crop and extract raster values in R</a>
 NOTE: The data used in this tutorial were collected at Harvard Forest which is
 a the National Ecological Observatory Network field site 
 <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
-More about the NEON Harvard Forest field site</a>. These data are proxy data for what will be
-available for 30 years from the NEON flux tower 
+More about the NEON Harvard Forest field site</a>. These data are a preview of the 30 year time-series that will be collected from the NEON flux tower and made available
```

Comment:
Hi @jsta preview and proxy are very different things. i'd like to stick with proxy for this section. They are a proxy for what will be collected at our sites - so not the same data but a parallel dataset.